modules/cluster-logging-release-notes-5.0.2.adoc

```diff
@@ -1,9 +1,34 @@
 [id="cluster-logging-release-notes-5-0-2"]
 = OpenShift Logging 5.0.2
 
-This release includes Red Hat Bug Advisory, link:https://access.redhat.com/errata/RHBA-2021:0963[RHBA-2021:0963 for OpenShift Logging Bug Fix Release (5.0.1)].
+This release includes Red Hat Bug Advisory, link:https://access.redhat.com/errata/RHBA-2021:1167[RHBA-2021:1167 for OpenShift Logging Bug Fix Release (5.0.2)].
 
 [id="openshift-logging-5-0-2-bug-fixes"]
 == Bug fixes
 
-* If you did not set `.proxy` in the cluster installation configuration and then configured a global proxy on the installed cluster, a bug prevented Fluentd from forwarding logs to Elasticsearch. To work around this issue, in the proxy/cluster configuration, set `no_proxy` to `.svc.cluster.local` so it skips internal traffic. The current release fixes the proxy configuration issue. Now, if you configure the global proxy after installing an {product-title} cluster, Fluentd forwards logs to Elasticsearch. (link:https://issues.redhat.com/browse/LOG-1187[*LOG-1187*])
+* If you did not set `.proxy` in the cluster installation configuration,
+and then configured a global proxy on the installed cluster, a bug
+prevented Fluentd from forwarding logs to Elasticsearch. To work around
+this issue, in the proxy/cluster configuration, set `no_proxy` to
+`.svc.cluster.local` so it skips internal traffic. The current release
+fixes the proxy configuration issue. Now, if you configure the global proxy
+after installing an OpenShift cluster, Fluentd forwards logs to
+Elasticsearch. (link:https://issues.redhat.com/browse/LOG-1187[*LOG-1187*])
+
+* Previously, forwarding logs to Kafka using chained certificates failed
+with error "state=error: certificate verify failed (unable to get local
+issuer certificate)." Logs could not be forwarded to a Kafka broker with a
+certificate signed by an intermediate CA. This happened because fluentd
+Kafka plugin could only handle a single CA certificate supplied in the
+ca-bundle.crt entry of the corresponding secret. The current release fixes
+this issue. It enables the fluentd Kafka plugin to handle multiple CA
+certificates supplied in the ca-bundle.crt entry of the corresponding
+secret. Now, logs can be forwarded to a Kafka broker with a certificate
+signed by an intermediate CA. (link:https://issues.redhat.com/browse/LOG-1216[*LOG-1216*], link:https://issues.redhat.com/browse/LOG-1218[*LOG-1218*])
+
+* Previously, an update in the cluster service version
+(CSV) accidentally introduced resource/limits for the OpenShift
+Elasticsearch operator container. Under specific conditions, this issue
+prevented the operator from being scheduled. The current release fixes this
```

Comment:
so this wasn't that it wouldn't be scheduled, it was that it would get OOM killed. it would be scheduled but then it'd die

Reply:
Updated. Thanks.